lib/modules/dosomething/dosomething_reportback/includes/dosomething_reportback.inc

```diff
@@ -199,19 +199,25 @@ class ReportbackEntity extends Entity {
    *
    * @param string $status
    *   The status of the reprotback file (approved, flagged, excluded, promoted)
-   * @param string $reason
-   *   Text to store in the flagged_reason property.
+   * @param array $values
+   *   Array storing values for flagged/promoted reasons.
    */
-  public function setFlagged($status = NULL, $reason = NULL) {
+  public function setFlagged($status = NULL, $values = NULL) {
```

Comment:
Should we rename this, since it sets Promoted as well as Flagged?  Maybe something like `setFlags` ?